themes/janeswalk/js/components/pages/Dashboard/DashboardResources.jsx

```diff
@@ -0,0 +1,101 @@
+import React from 'react';
+import DashboardStore from './DashboardStore';
+
+//TODO: Add styling, reference styling below, also City and Country Data is not available (assumed Toronto, Canada)
+
+const DashboardResources = () => {
+  const {cityOrganizers, videoTips, files, featuredWalks} = DashboardStore.getResources();
+  return (
+
+    <section>
+      <h2>Resources</h2>
+
+      <section className="walkOrganizers">
+        <h3>Connect with Fellow organizers</h3>
+        Got a question? Reach out to a fellow City Organizer for help.
+        <ul>
+          {cityOrganizers.map(co => (<li>{`${co.firstName} from ${co.cityName}: ${co.email}`}</li>))}
```

Comment:
We can keep for now, but look at the i18n translations with their `t()` functions in the future. We'll want to follow a standard `gettext` token system, so we can provide translations for these strings. e.g. this would take the form `t('%s from %s: %s', co.firstName, co.cityName, co.email)`